admin-proxy/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18-bullseye
```

Comment:
Do you need debian components?
I suggest using alpine image instead. Using alpine or light images has multiple advantages.
First: Security having the strict minimum of dependencies or program reduce the surface of attack.
Also using light image is a good practice for autoscalling. Light image can quickly be deployed.